### DISCUSSION TITLE
Error on Vercel

### DISCUSSION THREAD

**Discussion by PatelN123** [vercel/vercel#9282] [General] [answered]
I get this error when deoloying to vercel. Works fine locally.

Error: ENOENT: no such file or directory, copyfile '/var/task/node_modules/@vercel/remix/default-server.js' -> '/vercel/path0/build/__vc_handler.js'

**Comment by swarnava** [comment, marked as answer]
Hi,

This should be fixed now. Are you still facing issue?
